mongo/engine.py

```diff
@@ -390,6 +390,7 @@ class Submission(Document):
     code = ZipField(required=True, null=True, max_size=10**7)
     last_send = DateTimeField(db_field='lastSend', default=datetime.now)
     comment = FileField(default=None, null=True)
+    ipAddr = StringField()
```

Comment:
Please use `ip_addr` to keep the naming convention consistent. Maybe we should add some lint to check this automatically 🤔

Reply:
Fix